wandb/sdk/launch/runner/local.py

```diff
@@ -75,6 +75,8 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         validate_docker_installation()
         synchronous: bool = self.backend_config[PROJECT_SYNCHRONOUS]
         docker_args: Dict[str, Any] = self.backend_config[PROJECT_DOCKER_ARGS]
+        if launch_project.cuda is not None and launch_project.cuda:
```

Comment:
this should just be `if launch_project.cuda:`

Reply:
Fixed